vscode-dotnet-runtime-extension/package.json

```diff
@@ -67,6 +67,10 @@
 				"dotnetAcquisitionExtension.existingDotnetPath": {
 					"type": "array",
 					"description": "File Path to an existing installation of .NET."
+				},
+				"dotnetAcquisitionExtension.proxyUrl": {
+					"type": "string",
+					"description": "URL to a proxy if you use one, such as: https://proxy:port"
```

Comment:
clarification here: the goal is that users will not need to set this configuration, because the detection helpers you've added should do the Right Thing now, right? This is a fallback/backstop in case for some reason that doesn't work?

Reply:
Exactly the case :) Or if they have multiple proxies somehow and have a custom way of doing it.